receiver/mongodbreceiver/metadata.yaml

```
@@ -1 +1,143 @@
 name: mongodbreceiver
+
+attributes:
+  database:
+    description: The name of a database.
+  memory_type:
+    description: The type of memory used.
+    enum:
+      - resident
+      - virtual
+      - mapped
+      - mappedWithJournal
+  operation:
+    description: The mongoDB operation being counted.
+    enum:
+      - insert
+      - query
+      - update
+      - delete
+      - getmore
+      - command
+  connection_type:
+    description: The status of the connection.
+    enum:
+      - active
+      - available
+      - current
+  type:
+    description: The result of a cache request.
+    enum:
+      - hit
+      - miss
+
+metrics:
+  mongodb.cache.operations:
+    description: The number of cache operations of the instance.
+    unit: "{operations}"
+    enabled: true
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+    attributes: [type]
+  mongodb.collections:
+    description: The number of collections.
+    unit: "{collections}"
+    enabled: true
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [database]
+  mongodb.data.size:
+    description: The size in bytes of the collection. Data compression does not affect this value.
```

```suggestion
    description: The size of the collection. Data compression does not affect this value.
```
Units are specified in the units field